Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '3.0.1'
+ruby '2.7.4'
```

Comment:
2.7.4 has a trio of fairly serious CVEs against it. [I'd *strongly* recommend 2.7.5.](https://www.ruby-lang.org/en/news/2021/11/24/ruby-2-7-5-released/)

Reply:
FWIW the same CVEs applied to all previous versions in the 2.6 and 3.0 lines as well, so 3.0.1 is also vulnerable -- 3.0.3 is the patched version in the 3.0 line.

Reply:
Ah, just realised this isn't the .org repo, it's the blog 🤦

Reply:
thanks. I changed it to 2.7.5. It is related to this https://github.com/rubygems/rubygems.github.io/pull/78#issuecomment-983525018